src/pages/Editor.vue

```diff
@@ -156,6 +156,8 @@ import SliderWithMarkers from "@/components/UI/Slider/SliderWithMarkers.vue";
 import VideoPlayer from "@/components/UI/Player/VideoPlayer.vue";
 import ItemEditor from "@/components/Editor/ItemEditor.vue";
 import PlioService from "@/services/API/Plio.js";
+import ItemService from "@/services/API/Item.js";
+import QuestionService from "@/services/API/Question.js";
```

Comment:
I have changed this (what I am going to suggest) in my PR but I would recommend using ItemAPIService and QuestionAPIService and in general,  {Resource}{Utility}Service as the name everywhere as there could be other folders which have JS files with the same name - segregated by Resource.